sci-visualization/gwyddion/gwyddion-2.47.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit fdo-mime gnome2-utils python-single-r1
+
+DESCRIPTION="Framework for Scanning Mode Microscopy data analysis"
+HOMEPAGE="http://gwyddion.net/"
+SRC_URI="http://gwyddion.net/download/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc fits fftw gnome kde nls opengl perl python ruby sourceview xml X"
+
+RDEPEND="
+	media-libs/libpng:0
+	x11-libs/cairo
+	x11-libs/gtk+:2
+	x11-libs/libXmu
+	x11-libs/pango
+	fits? ( sci-libs/cfitsio )
+	fftw? ( sci-libs/fftw:3.0 )
+	gnome? ( gnome-base/gconf:2 )
+	kde? ( kde-frameworks/kdelibs:4 )
+	opengl? ( virtual/opengl x11-libs/gtkglext )
+	perl? ( dev-lang/perl )
```

Comment:
so interfacing perl with normal build systems is messy. This is the wrong path (for which you're ofc not responsible). In general, packages installing `.pm`'s need to install them into Perl's `$VENDORLIB`, and that depends on Perl's subslot, i.e.:

1. You need to `:=` depend on perl
2. The `.pm`'s need to be installed correctly

Installing `.pm`'s correctly from non-Perl build systems is a slight nightmare at the moment, but @kentfredric has submitted a stack of helper functions to the ml to make this stuff half-way sane and in the spirit of `insinto`/`doins`. Once that stuff is merged, we can proceed here.